go.mod

```diff
@@ -462,10 +462,11 @@ replace (
 // check for retracted versions: go list -mod=mod -f '{{if .Retracted}}{{.}}{{end}}' -u -m all
 replace github.com/microcosm-cc/bluemonday => github.com/microcosm-cc/bluemonday v1.0.24
 
-// Uncomment for local development for testing with changes in the components-contrib repository.
+// Uncomment for local development for testing with changes in the components-contrib && kit repositories.
```

Comment:
From a new contributor perspective I think it would be nice to have kit mentioned here in the off chance that a user is testing a change in kit. It is more explicit and easier on developers to uncomment a line rather than add it in. Not sure how we can disable the failing CI check on me adding this in?

Reply:
It's here: https://github.com/dapr/dapr/blob/master/.github/scripts/check_go_mod.mjs

Reply:
I'll pull this into a separate PR 👍🏻